components/brave_wallet_ui/components/desktop/lock-screen/index.tsx

```diff
@@ -99,6 +102,33 @@ export const LockScreen = () => {
 
   const isAndroid = loadTimeData.getBoolean('isAndroid') || false
 
+  if (isAndroid) {
+    // Methods
+    const onDoubleTap = (e: React.MouseEvent<HTMLButtonElement>) => {
+      if (e.detail === 2) {
+        getWalletAPIProxy().pageHandler?.unlockWalletUI()
+      }
+    }
+
+    return (
+      <AndroidLockScreenWrapper onClick={onDoubleTap}>
+        <DoubleTapIcon />
+        <Text
+          textSize='22px'
+          textColor='primary'
+        >
+          {getLocale('braveWalletDoubleTapScreen')}
+        </Text>
+        <Text
+          textSize='16px'
+          textColor='tertiary'
+        >
+          {getLocale('braveWalletUnlockAndroidDescription')}
+        </Text>
+      </AndroidLockScreenWrapper>
+    )
+  }
```

Comment:
nit: Can we move this under the `// render` section?

Reply:
Moved in https://github.com/brave/brave-core/pull/24992/commits/7bb8853179aa94459eddd166de6aecfc376cb0ce